### ISSUE TITLE
libdnet problems

### ISSUE THREAD

**Issue by awaldow** [DataSoft/Honeyd#23]
Got lucky trying to get honeyd to work on my home machine; I had snort on here from a while ago, and getting libdnet was part of the deal. There are lines in the configure file that make it seems that honeyd supported libdnet once, but I kept getting 'undefined references' to libdumbnet1 functions because the configure process will pick libdnet over libdumbnet every time if it's available. Which was weird, because I know that libdnet contains some of the functions that were getting undefined reference errors, yet it seemed to be looking for libdumbnet analogs.

In short, it appears that if libdnet and libdumbnet are both on the machine, undefined references result from the ./configure step. It may have been a local circumstance, but should be tested anyways.

**Comment by awaldow** [comment]
To add more information, this was on Ubuntu 12.04, upgraded from 11.04. May have been some incompatibility stemming from an out-of-date version of libdnet in the 11.04 repos or something, though that's pure conjecture.

**Comment by awaldow** [comment]
Just a problem of libdnet being out of date.
